yo/db.py

```diff
@@ -90,47 +88,47 @@ class Priority(enum.Enum):
 notifications_table = sa.Table(
     'yo_notifications',
     metadata,
-    sa.Column(
-        'nid',
-        sa.String(32),
-        primary_key=True,
-        default=lambda: uuid.uuid4().hex),
+    sa.Column('nid', sa.String(36), primary_key=True),
     sa.Column('notify_type', sa.String(20), nullable=False, index=True),
     sa.Column('to_username', sa.String(20), nullable=False, index=True),
     sa.Column('from_username', sa.String(20), index=True, nullable=True),
-    sa.Column('json_data', sa.UnicodeText),
+    sa.Column('json_data', sa.UnicodeText(1024)),
     sa.Column(
         'created',
         sa.DateTime,
         default=sa.func.now(),
         nullable=False,
         index=True),
+    sa.Column(
+        'updated',
+        sa.DateTime,
+        default=sa.func.now(),
```

Comment:
Shouldn't updated be set only when the value is updated? I don't think we should use two different SQL datetime functions.  Decide on either `now` or `current_timestamp` (they are synonymous) and be consistent

Reply:
Using sa.func.now() for both, the default needs to be set to something as well and it makes logical sense for the last update to be the time something was created, hence the default.